mysql-test/main/mysql_json_mysql_upgrade_system_tables.test

```diff
@@ -0,0 +1,39 @@
+--echo #
+--echo # MDEV-32462: mysql_upgrade -s still checks for non system tables
+--echo #
+
+# Let's now load plugin first
+-- source include/have_utf8.inc
+# Don't load plugin in order to get ER_UNKNOWN_DATA_TYPE instead of ER_TABLE_NEEDS_REBUILD
```

Comment:
To prevent such a requirement:
```
--error ER_UNKNOWN_DATA_TYPE,ER_TABLE_NEEDS_REBUILD
```

This would also make the `select from mysql.plugin` unreliable, but I'm not sure that's a significant part of this test.

Reply:
Yes, not needed `select` will remove.